db/seeds.rb

```diff
@@ -1,9 +1,14 @@
-address = "Louvain La Neuve"
-capacity = 9
-duration = 90
+if Rails.env.development?
+  Event.destroy_all
 
-for i in 1..6 do
-  time = i * 100
-  start_time = time.hours.from_now
-  Session.create(start_time:, duration:, address:, capacity:)
+  address = "Louvain La Neuve"
+  capacity = 9
+  duration = 90
+  price = 12
+
+  for i in 1..6 do
+    time = i * 100
+    start_time = time.hours.from_now
+    Event.create(start_time:, duration:, address:, capacity:, price:)
```

Comment:
I'm talking about price: as opposed to :price_cents. I'm assuming we aren't going to be working with floats, just whole numbers.

Reply:
money-rails convert price to price_cents
It was said in the video that when price is an integer it it safe to use and when the price is a float like 12.5 for example then it's better to write the price in cents instead to avoid incorrect rounding